vendors/pc/manifest.cmake

```diff
@@ -1,6 +1,6 @@
 set(
     AFR_MANIFEST_SUPPORTED_BOARDS
-    windows
+    ""
```

Comment:
Windows is supported now.

Reply:
There were no CMakefiles in my revision.. I can update it in later version